docs/visual-basic/language-reference/operators/null-conditional-operators.md

```diff
@@ -31,6 +31,8 @@ For comparison, the alternative code for the first of these expressions without
 Dim length As Integer
```

Comment:
Should this be a nullable to match the example on line 19?

```suggestion
Dim length As Integer?
```

Reply:
Thanks!